modules/developers-guide/pages/rust-developing.adoc

```diff
@@ -0,0 +1,92 @@
+== Rust developing
+
+=== Building
+
+.Install Rust toolchain:
+
+[source, shell]
+----
+include::example$bash/rust-build1.sh[]
+----
+
+.Run build in the root directory of the project:
+
+[source, shell]
+----
+include::example$bash/rust-build2.sh[]
+----
+
+=== Running the examples
+
+For your convenience, this project contains a bunch of examples located in the
+examples directory, which demonstrate connecting, creating schemas,
+inserting data and querying.
+You'll need a working instance of a Stargate cluster to run the examples.
+Refer to the official link:https://stargate.io/docs/[Stargate documentation]
+for more details on how to setup Stargate.
+
+Each example program accepts an URL of the Stargate coordinator,
+the authentication token and the keyspace name:
+
+[source, rust]
+----
+include::example$rust/rust-run-example.rs[]
+----
+
+The authentication token value can be also defined in the SG_TOKEN environment variable.
+
+Here is one example of starting a Cassandra cluster and launching Stargate using
```

Comment:
As mentioned above I've opened a PR to remove ccm.

Reply:
Thanks - I've made the changes to the docs, too.